CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,13 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2025-07-23 Add spell checking in Notes View
+<p>Spell check now added to following functions:
+<p>Notes View -> New Note Pane
+<p>Conversation View -> Translation Conversation Contribution Provider
+<p>Import/Export -> New Attribute Dialog
+<p>Attribute Editor -> Transaction
```

Comment:
It might be worth altering some of the wording here to be more user-friendly. e.g. I'm not sure whether a user will understand what a 'Translation Conversation Contribution Provider' is. 
Also for the Attribute Editor, I believe it should be all string attributes instead of just Transaction?

Reply:
updated